perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/SecurityTeamsManagerBlImpl.java

```diff
@@ -111,19 +111,23 @@ public void deleteSecurityTeam(PerunSession sess, SecurityTeam securityTeam, boo
 		if (blacklist != null && !blacklist.isEmpty() && !forceDelete) {
 			throw new RelationExistsException("SecurityTeam has blacklisted users.");
 		}
-		for (User blacklistedUser : blacklist) {
```

Comment:
In this case, the blackList cannot be null. The method above never returns null. Actually, the problem is in the if statement above. `blacklist != null` should be removed.